test/spec/modules/permutiveRtdProvider_spec.js

```diff
@@ -42,10 +42,17 @@ describe('permutiveRtdProvider', function () {
         return { id: seg }
       })
 
+      const expectedBidderExtensionData = {
+        ix: {
+          segtax: '4'
```

Comment:
(learning opportunity) Would you able to link (in a reply) to the resources which indicate what effect this configuration has?

Reply:
Pages 10 and 11 from the [IAB Taxonomy documentation](https://iabtechlab.com/wp-content/uploads/2022/02/IAB-Tech-Lab-Taxonomy-and-Data-Transparency-Standards-to-Support-Seller-defined-Audience-and-Context-Signaling.pdf) explain it. 

My rough understanding is that it's a way to indicate which taxonomy to use, where a taxonomy is a collection of segments. So this is a way to indicate to bidders that the segment IDs in the request will correspond to the [new IAB audience taxonomy](https://iabtechlab.com/standards/audience-taxonomy/).